Samples/Samples/View/CalendarPage.xaml

```diff
@@ -1,49 +1,61 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<views:BasePage xmlns="http://xamarin.com/schemas/2014/forms"
+<views:BasePage  xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:views="clr-namespace:Samples.View"
              xmlns:viewmodels="clr-namespace:Samples.ViewModel"
              xmlns:essentials="clr-namespace:Xamarin.Essentials;assembly=Xamarin.Essentials"
              xmlns:samples="clr-namespace:Samples.ViewModel;assembly=Samples"
              x:Class="Samples.View.CalendarPage"
-             Title="Calendar">
-    <views:BasePage.BindingContext>
-        <viewmodels:CalendarViewModel />
-    </views:BasePage.BindingContext>
-    <views:BasePage.Resources>
-        <ResourceDictionary>
-            <DataTemplate x:Key="CalendarItemTemplate">
-                <ViewCell>
-                    <StackLayout>
-                        <Label Text="{Binding Name}" TextColor="Black" HorizontalTextAlignment="Center"/>
-                    </StackLayout>
-                </ViewCell>
-            </DataTemplate>
-        </ResourceDictionary>
+             Title="Calendars">
```

Comment:
Probably not essential but I would suggest defining `x:DataType` at the page level and in the `DateTemplate`s